### ISSUE TITLE
Typo in test_irc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6392]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)| @lvh reported|
|-|-|
|Trac ID|trac#6392|
|Type|enhancement|
|Created|2013-03-26 17:40:46Z|
|Branch|https://github.com/twisted/twisted/tree/typo-in-test_irc-6392|

17:24 < Taggnostr> I found a typo in http://twistedmatrix.com/trac/browser/trunk/twisted/words/test/test_irc.py#L1689
17:24 < Taggnostr> s/IRCCLient/IRCClient/ (lowercase l)

Attachments:

* [my-twisted-patch.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/892/892dfab98fd8d7579462a5192e8666d9b7308922/d5bf14bfe4ce8fc7b9256a069dbe396aeaef28e5.patch) (517 bytes) - added by Infinity on 2013-03-27 18:23:03Z - Patch-v0 : Fixed the Typo (Changed 'L' to 'l')

<details><summary>Searchable metadata</summary>

```
trac-id__6392 6392
type__enhancement enhancement
reporter__lvh lvh
priority__normal normal
milestone__ 
branch__branches_typo_in_test_irc_6392 branches/typo-in-test_irc-6392
branch_author__tomprince tomprince
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy_typo easy typo
time__1364319646000000 1364319646000000
changetime__1364411568000000 1364411568000000
version__None None
owner__tom_prince tom.prince

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity set owner to Infinity<br>Infinity set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Infinity removed owner<br>Infinity set status to `new`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_3"></a>|@tomprince commented|
|-|-|

(In [37846]) Branching to typo-in-test_irc-6392.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_4"></a>|@tomprince commented|
|-|-|

(In [37847]) Apply my-twisted-patch.patch from Infinity.

Refs: [#6392](https://github.com/twisted/twisted/issues/6392)

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_5"></a>|@tomprince set owner to @tomprince|
|-|-|

I added a [.misc newsfile](https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles).

Otherwise, this change looks good. I'll merge.

**Comment by twisted-trac** [comment]
|<img alt="Infinity's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|Infinity commented|
|-|-|

Thanks for adding the newsfile and for the quick merge :)

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_7"></a>|@tomprince set status to `closed`|
|-|-|

(In [37850]) Merge typo-in-test_irc-6392: Fix typo in docstring in twisted.words.test.test_irc.

Author: Infinity
Reviewers: tom.prince
Fixes: [#6392](https://github.com/twisted/twisted/issues/6392)
